include/exadg/acoustic_conservation_equations/user_interface/boundary_descriptor.h

```diff
@@ -44,13 +44,17 @@ namespace Acoustics
  * | prescribe velocity values |                         | Dirichlet:              |
  * |                           | no BCs to be prescribed | prescribe g_u           |
  * +---------------------------+-------------------------+-------------------------+
+ * | admittance BC             |                         | Admittance:             |
+ * |                           | no BCs to be prescribed | prescribe Y             |
+ * +---------------------------+-------------------------+-------------------------+
```

Comment:
I think somewhere here the equation provided in the description of this PR needs to be written for documentation. Otherwise, it is not clear to the user where the factor `rho` and `c` appear and whether `Y` already accounts for them or not. Please also note comment https://github.com/exadg/exadg/pull/629/files#r1450469079 in this context.

Reply:
[b81837d](https://github.com/exadg/exadg/pull/629/commits/b81837d0c9fa450f3a3ed37d6051e77712891b91)